### PR TITLE
Add config.yml note

### PR DESCRIPTION
**Pull request by JasonHorkles**
Added a note about setting a bed home in the day for 1.15+ because @triagonal [said so](https://discordapp.com/channels/390942438061113344/426460575971672064/718609253337923675)

### CODE REVIEW

**Comment by pop4959** [comment]
Have you tested it? As far as I know, the only thing that changed with 1.15 was that the bed spawn message is no longer handled by Essentials. I don't think setting spawn during the day got removed.

**Comment by JasonHorkles** [comment]
Ik the sethome.bed perm no longer works so idk why this would
Was mostly going off what triangle said though haha

**Comment by triagonal** [comment]
Just tested quickly. Looks like the player's bed respawn point is still set by Minecraft even if `update-bed-at-daytime` is set to `false` in config, meaning that the entire feature is indeed ineffective on Minecraft 1.15 and above.

**Comment by pop4959** [comment]
OK. Maybe put that explanation as part of the comment, so people aren't confused?

File: Essentials/src/config.yml [hunk 1]
```diff
@@ -621,6 +621,7 @@ max-projectile-speed: 8
 ############################################################
 
 # Allows people to set their bed during the day.
+# Has no effect in Minecraft 1.15+
```
Comment (triagonal) [inline Essentials/src/config.yml]:
```suggestion
# This setting has no effect in Minecraft 1.15+, as Minecraft will always allow the player to set their bed location during the day.
```

**Review by pop4959 — APPROVED** [review]

**Review by mdcfe — APPROVED** [review]

**Pull request merged**